subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc

```diff
@@ -206,13 +206,13 @@ There are three ways to compare builds in Gradle Enterprise.
 
 1. Use the scan list to find the scans to compare, selecting them by clicking the respective image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] or image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon.
 2. From a scan that you wish to compare with another, click the image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] icon at the top of the scan, then select the second scan by clicking the  image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon from the list of scans.
-3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a build scan to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
+3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a Build Scan™ to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
 
```

Comment:
Ugh, this section is full of references to just "scan" too and very outdated. (2) and (3) are completely different now.

Can we delete most of this and link to https://docs.gradle.com/enterprise/tutorials/task-inputs-comparison/?